net/macos-fortress/Portfile

```diff
@@ -0,0 +1,817 @@
+    }
+    # create or initialize the configfile
+    if { ![file exists ${configfile}] || [variant_isset "initialize_always"] } {
+        delete ${configfile}
+        if { [file exists ${configdefault}] } {
+            xinstall -m 0644 ${configdefault} ${configfile}
+        } elseif { [file exists ${configfile}.previous] } {
+            xinstall -m 0644 ${configfile}.previous ${configfile}
+        }
+    }
+    # patch the configfile if it's the default
+    if { [portchecksum::calc_rmd160 ${configfile}] 
+        eq [portchecksum::calc_rmd160 ${configdefault}] } {
+        system \
+            "${patch.cmd} ${patch.pre_args} \
+                -f -l -N -r /dev/null ${configfile} < ${patchfile}"
+    } else {
+        ui_warn "File ${configfile} is not the default
+\tand *not* being patched."
+    }
+}
+
+proc plutil_startup {plcmds label} {
+    global prefix startupitem.location
+    foreach cmd ${plcmds} {
+        system -W ${prefix}/etc/${startupitem.location}/${label} \
+            "/usr/bin/plutil ${cmd} ${label}.plist"
+    }
+}
+
+set notes_pf        "\
+The PF configuration provides an adaptive firewall that blocks brute
+force attacks, and connections from IP addresses provided by the
+crowd-sourced lists dshield and emergingthreats. PF uses this
+environment variable (with default value):
+
+\t\${PF_CONF:-${pf_conf}}
+"
+
+set notes_proxy     "\
+The proxy uses a chain of squid (port 3128) and privoxy (port 8118)
+along with a blackhole provided by nginx (port 8119). Domain names
+provided by hphosts and a blacklist file are blocked, excluding
+whitelisted domain names. These are provised in the files:
+
+\t${prefix}/etc/${name}/blacklist.txt
+\t${prefix}/etc/${name}/whitelist.txt
+
+The proxy also provides a proxy autoconfiguration (PAC) file with
+blocking rules generated from easylist ad and tracker blocks. The
+proxy uses these environment variables (with default values):
+
+\t\${PROXY_HOSTNAME:-${proxy_hostname}}
+\t\${PROXY_PAC_SERVER:-${proxy_pac_server}}
+\t\${PROXY_PAC_DIRECTORY:-${proxy_pac_directory}}
+
+The native macOS web server is used by default to host the PAC file.
+This web server must be launched independently with the command
+
+sudo apachectl start
+
+Clients may be configured to use this proxy by either host:port or
+the PAC file:
+
+\t${proxy_hostname}:3128
+\thttp://${proxy_hostname}/proxy.pac
+"
+
+if {${subport} eq ${name}} {
+    description     Firewall, Blackhole, and Privatizing Proxy for Trackers, Attackers, Malware, Adware, and Spammers
+    long_description    \
+        Kernel-level, OS-level, and client-level security for macOS. Built\
+        to block attacks using open source databases, and block ads,\
+        malicious scripts, and conceal information used for web tracking.\
+        Uses PF, squid, privoxy, hphosts, dshield, emergingthreats,\
+        hostsfile, and a proxy autoconfiguration (PAC) file.
+
+    homepage        https://github.com/essandess/macOS-Fortress
```

Comment:
Is there a reason that the GitHub portgroup is not being used?

Reply:
It’s not necessary for the main port because the Portfile itself takes care of the install and configuration. There’s nothing to download from the parent repo.